src/udiskslinuxdrive.c

```diff
@@ -40,6 +40,7 @@
 #include <mntent.h>
 
 #include <glib/gstdio.h>
+#include <libstoragemgmt/libstoragemgmt.h>
```

Comment:
No, please don't bury this PR. The goal is to support enterprise deployments as well. But mainly with the modules not with the core daemon. We need to work with other distributions and get as wide adoption as possible. Adding hard dependencies is a problem in that direction. Adding things not focusing on laptops and/or workstations is great, it just needs to go to the modules.